doc/chapters/api/index.rst

```diff
@@ -0,0 +1,111 @@
+
+.. _api:
+
+API
+***
+
+This chapter documents the DBus and C API for interacting with the SoftwareContainerAgent. This is
+mostly useful for integrators.
+
+.. TODO::
+    This chapter is still to be written properly. Some bits and pieces have
+    been but they are under development.
+
+.. _dbus-api:
+
+DBus API
+========
+
+:Path: /com/pelagicore/SoftwareContainer
+:Interface: com.pelagicore.SoftwareContainerAgent
+
+Methods
+-------
+
+Ping
+----
+Ping()
```

Comment:
I was thinking that the lines that are code could be a different format. Have you considered this?